dotcom-rendering/src/web/components/PullQuoteBlockComponent.tsx

```diff
@@ -112,19 +125,65 @@ function decideFont(role: string) {
 export const PullQuoteBlockComponent: React.FC<{
 	html?: string;
 	palette: Palette;
-	design: ArticleDesign;
+	format: ArticleFormat;
 	role: string;
 	attribution?: string;
-}> = ({ html, palette, design, attribution, role }) => {
+}> = ({ html, palette, format, attribution, role }) => {
 	if (!html) return <></>;
-	switch (design) {
+
+	if (format.theme === ArticleSpecial.SpecialReportAlt)
```

Comment:
Maybe I missed something, but if we're returning here, do we still need the 'decideFontWeight' use below?

Reply:
You're right, I removed it